nautilus_open_any_terminal/locale/Makefile

```diff
@@ -0,0 +1,22 @@
+LANGUAGES := de es fr pl pt_BR ru sw tr zh_CN
```

Comment:
would it be possible to not have this hard-coded here? I will forget it for the next language.